prog.go

```diff
@@ -631,3 +643,18 @@ func resolveBTFType(name string, progType ProgramType, attachType AttachType) (b
 		return nil, nil
 	}
 }
+
+var haveBpfLSM = internal.FeatureTest("LSM", "5.8", func() (bool, error) {
+	lsm, err := ioutil.ReadFile("/sys/kernel/security/lsm")
+	if err != nil {
+		return false, fmt.Errorf("LSM unavailable: %w", ErrNotSupported)
+	}
+
+	for _, module := range strings.Split(string(lsm), ",") {
+		if module == "bpf" {
+			return true, nil
+		}
+	}
+
+	return false, fmt.Errorf("BPF LSM not loaded: %w", ErrNotSupported)
```

Comment:
I think at this point we know that the feature is not available.

```suggestion
	return false, nil
```